docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -303,6 +325,53 @@ image::images/fleet-policy-hidden-secret.png[Screen capture showing a hidden sec
 
 . Click **Save integration**. The original secret value is overwritten in the policy.
 
+[discrete]
+[[agent-policy-limit-cpu]]
+== Set the maximum CPU usage
+
+You can limit the amount of CPU consumed by {agent}. This parameter limits the number of operating system threads that can be executing Go code simultaneously in each Go process. You can specify an integer value not less than `0`, which is the default value that stands for "all available CPUs".
+
+This limit applies independently to the agent and each underlying Go process that it supervises. For example, if {agent} is configured to supervise two {beats} with a CPU usage limit of `2` set in the policy, then the total CPU limit is six, where each of the three processes (one {agent} and two {beats}) may execute independently on two CPUs.
+
+This setting is similar to the {beats} {filebeat-ref}/configuration-general-options.html#_max_procs[`max_procs`] setting. For more detail, refer to the link:https://pkg.go.dev/runtime#GOMAXPROCS[GOMAXPROCS] function in the Go runtime documentation.
+
+. In {fleet}, click **Agent policies**.
+Select the name of the policy you want to edit.
+
+. Click the **Settings** tab and scroll to **Advanced settings**.
+
+. Set **Limit CPU usage** as needed. For example, to limit Go processes supervised by {agent} to two operating system threads each, set this value to `2`.
+
+[discrete]
+[[agent-policy-http-monitoring]]
+== Override the default monitoring port
+
+You can override the default port that {agent} uses to send monitoring data. It's useful to be able to adjust this setting if you have an application running on the machine on which the agent is deployed, and that is using the same port.
+
+. In {fleet}, click **Agent policies**.
+Select the name of the policy you want to edit.
+
+. Click the **Settings** tab and scroll to **Advanced settings**.
+
+. Set **Agent HTTP monitoring** setting to enabled, and then specify a host and port for the monitoring data output.
+
+. Enable **buffer.enabled** if you'd like {agent} and {beats} to collect metrics into an in-memory buffer and expose these through a `/buffer` endpoint. This data can be useful for debugging or if the {agent} has issues communicating with {es}. Enabling this option may slightly increase process memory usage.
```

Comment:
Currently the `enabled` and `buffer.enabled` flags are not visible on the UI because I couldn't test it successfully: https://github.com/elastic/kibana/pull/180922#issuecomment-2110007599
@ycombinator Should I raise an issue on agent side for this or can someone check on agent side if these flags can be changed from the policy?

Reply:
@juliaElastic  For now I've removed mention of the `enabled` and the `buffer.enabled` flags. If these get added back in please just let me know and I'll fix up the docs.